src/Microsoft.ML.Data/DataLoadSave/CompositeDataReader.cs

```diff
@@ -12,19 +12,19 @@ namespace Microsoft.ML.Data
     /// This class represents a data reader that applies a transformer chain after reading.
     /// It also has methods to save itself to a repository.
     /// </summary>
-    public sealed class CompositeDataReader<TSource, TLastTransformer> : IDataReader<TSource>
+    public sealed class CompositeDataReader<TSource, TLastTransformer> : IDataLoader<TSource>
         where TLastTransformer : class, ITransformer
     {
         /// <summary>
         /// The underlying data reader.
         /// </summary>
-        public readonly IDataReader<TSource> Reader;
+        public readonly IDataLoader<TSource> Reader;
```

Comment:
```suggestion
        public readonly IDataLoader<TSource> Loader;
```
Please double-check other public fields and public functions' arguments.

Reply:
Also update the comment here.

---
In reply to: [260381373](https://github.com/dotnet/machinelearning/pull/2731#discussion_r260381373) [](ancestors = 260381373)